code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -437,6 +437,17 @@
 		else
 			add_mood_event("tailpulled", /datum/mood_event/tailpulled)
 
+	else if ((helper.zone_selected == BODY_ZONE_PRECISE_GROIN) && (istype(head, /obj/item/clothing/head/costume/kitty) || istype(head, /obj/item/clothing/head/collectable/kitty)))
```

Comment:
Technically `else if` is invalid syntax that Byond helpfully interprets correctly anyway, would it be possible for you to also refactor this proc to use early returns instead?
And by invalid syntax I mean that
```
else if
```
Is invalid while
```
else
	if
```
is valid

Reply:
`else if` is valid syntax and it's used extensively in the codebase.

![Code_DVEomzuufA](https://user-images.githubusercontent.com/5195984/197819746-9a2d5613-0c21-4c09-b751-29fae3400c65.png)

Reply:
> `else if` is valid syntax and it's used extensively in the codebase.
> 
> ![Code_DVEomzuufA](https://user-images.githubusercontent.com/5195984/197819746-9a2d5613-0c21-4c09-b751-29fae3400c65.png)

According to Ike it's not valid syntax it's just that the compiler is lenient about it

Reply:
It's literally in the DM docs: https://www.byond.com/docs/ref/#/proc/if

Reply:
> According to Ike it's not valid syntax it's just that the compiler is lenient about it

Who is Ike? `else if` loops are super common and any person working with our codebase should be familiar with them.

Reply:
> > According to Ike it's not valid syntax it's just that the compiler is lenient about it
> 
> Who is Ike? `else if` loops are super common and any person working with our codebase should be familiar with them.

Main person coding OpenDream basically

Reply:
Regardless, I still think that this proc should have early returns in it and not be a chiant if else chain

Reply:
i'm fixing that